### ISSUE TITLE
[Feature Request] HDMI-CEC compoatiblity

### ISSUE THREAD

**Issue by Symbianx** [MediaBrowser/Emby.Chromecast#15]
Hello,
It would be really nice if you added HDMI-CEC support on chromecast, it's really a bummer to use the phone to pause a movie. The chromecast supports it for other apps and the stop button somewhat works (stops the movie and closes the emby app).

Thank you

**Comment by LukePulverenti** [comment]
Hi, so you're saying you can stop from a CEC remote, and that's it?

**Comment by Symbianx** [comment]
Yes Luke, only the stop button works, but according to battamo that's by default.

**Comment by jmunowitch** [comment]
I can confirm this issue as well. Currently the Pause/Play is not being passed to the Chromecast through CEC.

**Comment by LukePulverenti** [comment]
We've updated our Chromecast receiver. Please try again. Thanks !
